xarray/tests/test_combine.py

```diff
@@ -1163,16 +1166,3 @@ def test_combine_by_coords_raises_for_differing_types():
         TypeError, match=r"Cannot combine along dimension 'time' with mixed types."
     ):
         combine_by_coords([da_1, da_2])
-
-
-def test_combine_by_coords_raises_for_no_index():
```

Comment:
FWIW this was a nice test because it included the coord name! But no great stress